tests/model_fields/models.py

```diff
@@ -369,6 +369,10 @@ class PrimaryKeyUUIDModel(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
 
 
+class RelatedToUUIDModel(models.Model):
+    rel = models.ForeignKey('PrimaryKeyUUIDModel')
```

Comment:
A name like `uuid_pk` might avoid confusion with `field.rel` but no worries if you think the current name is fine.